unistd/file.c

```diff
@@ -41,6 +41,8 @@ extern int sys_lseek(int fildes, off_t *offset, int whence);
 
 WRAP_ERRNO_DEF(ssize_t, read, (int fildes, void *buf, size_t nbyte), (fildes, buf, nbyte))
 WRAP_ERRNO_DEF(ssize_t, write, (int fildes, const void *buf, size_t nbyte), (fildes, buf, nbyte))
+WRAP_ERRNO_DEF(ssize_t, pread, (int fildes, void *buf, size_t nbyte, off_t offset), (fildes, buf, nbyte, offset))
+WRAP_ERRNO_DEF(ssize_t, pwrite, (int fildes, const void *buf, size_t nbyte, off_t offset), (fildes, buf, nbyte, offset))
```

Comment:
If both `pread`/`pwrite` and `read`/`write` will use one syscall, then you should check for negative `offset` in `pread`/`pwrite` wrapper here (and return EINVAL), because it will be impossible to discriminate `p` and non`p` functions at the syscall level and POSIX requires `p` functions to reject negative `offset`.